lib/src/map/flutter_map_state.dart

```diff
@@ -87,50 +124,50 @@ class FlutterMapState extends MapGestureMixin {
             onLongPress: handleLongPress,
             onDoubleTap: handleDoubleTap,
             child: GestureDetector(
-              onScaleStart: handleScaleStart,
-              onScaleUpdate: handleScaleUpdate,
-              onScaleEnd: handleScaleEnd,
               onTap: _positionedTapController.onTap,
               onLongPress: _positionedTapController.onLongPress,
               onTapDown: _positionedTapController.onTapDown,
               onTapUp: handleOnTapUp,
-              child: ClipRect(
-                child: Stack(
-                  children: [
-                    OverflowBox(
-                      minWidth: size.x,
-                      maxWidth: size.x,
-                      minHeight: size.y,
-                      maxHeight: size.y,
-                      child: Transform.rotate(
-                        angle: mapState.rotationRad,
-                        child: Stack(
-                          children: [
-                            if (widget.children != null &&
-                                widget.children.isNotEmpty)
-                              ...widget.children,
-                            if (widget.layers != null &&
-                                widget.layers.isNotEmpty)
-                              ...widget.layers.map(
-                                (layer) => _createLayer(layer, options.plugins),
-                              )
-                          ],
+              child: scaleGestureDetector(
```

Comment:
If I understand this correctly @ritikmishra, the TeamGestureRecognizer will always ensure that flutter_map takes precedence over ListView scoll? I don't think we can conclude that this is the only behaviour user expect. I think we need to make this behavior configurable, just as it is for [scollables](https://api.flutter.dev/flutter/widgets/Scrollable-class.html) in general. I think we should add something similar (or equal) to [scrollPhysics](https://api.flutter.dev/flutter/widgets/ScrollPhysics-class.html) to MapOptions constructor.

@johnpryan? What do you think?